tests/Integration/PrestaShopBundle/Controller/Api/Improve/Design/PositionsControllerTest.php

```diff
@@ -125,12 +155,12 @@ public function testMoveHookPositionToTop()
                 'api_improve_design_positions_update'
             ),
             [
-                'moduleId' => 4,
-                'hookId' => 3,
+                'moduleId' => $this->moduleId,
+                'hookId' => $this->hookId,
                 'way' => 0,
                 'positions' => [
-                    '65_4',
-                    '65_3',
+                    sprintf('%d_%d', $this->moduleId, $this->hookId),
```

Comment:
I don't understand the format of this query? do provide the former and new position of one module?
or do you need to provide the former and new module for one position?

Reply:
I don't understand too. The aim of the migration was to keep to old behavior, even for Api :nauseated_face:

Reply:
sad..